components/calendar/Cargo.toml

```diff
@@ -46,10 +46,21 @@ serde = { version = "1.0", default-features = false, features = ["derive", "allo
 zerovec = { version = "0.7", path = "../../utils/zerovec", default-features = false, features = ["derive"] }
 
 [dev-dependencies]
+criterion = "0.3"
 icu = { path = "../icu", default-features = false }
 icu_benchmark_macros = { version = "0.6", path = "../../tools/benchmark/macros" }
-icu_calendar = { version = "0.6", path = "../calendar", features = ["serde"] }
+icu_calendar = { version = "0.6", path = "../calendar", features = ["serde"] } # Dependency required to prevent `delayed_good_path_bugs` error (https://github.com/unicode-org/icu4x/pull/1844#issuecomment-1118111564)
```

Comment:
Question: am I correct in understanding that this "bug" happens when compiling code when the right impls are not available to build it? If so, a cleaner solution would be to add more `required-features` gates to the appropriate files.

Reply:
The compilation error triggers for the line at https://github.com/unicode-org/icu4x/blob/main/components/calendar/src/japanese.rs#L17. From https://github.com/unicode-org/icu4x/blob/main/components/calendar/Cargo.toml#L33, I would believe that the correct features are being implemented, but I could be mistaken. A few stack traces are output, so it's challenging to post the full here

Reply:
Please remove the comment now that the issue appears to have been fixed?